decidim-core/app/helpers/decidim/application_helper.rb

```diff
@@ -3,6 +3,9 @@
 module Decidim
   # Main module to add application-wide helpers.
   module ApplicationHelper
+    # Include decidim-comments helpers so they can be used anywhere
+    include Decidim::Comments::CommentsHelper
```

Comment:
If you do, which is still not clear, remember to add `decidim-comments` as a dependency of pages and remove it from `core`.